### PR TITLE
fix delete behavior on text cells.

### LINKED ISSUE #12878: WYSIWYG Select all text and press delete key twice shortens the size of the cell
- Azure Data Studio Version:
Version: 1.23.0-rc1
Commit: 7429407029af2e08d17a8928825815cffad94f17
Date: 2020-10-09T00:40:51.445Z
VS Code: 1.49.0
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0

Steps to Reproduce:

1. Create a text cell and type something
2. Cmd + A and then press the delete key twice
3. The size of the cell is reduced by half.
Screen Shot 2020-10-09 at 3 49 35 PM

### PR DESCRIPTION
**Pull request by MaddyDev**
Remove hardcoded default cell content  and add css instead.

This PR fixes #12878

### CODE REVIEW

**Comment by chlafreniere** [comment]
@MaddyDev I believe the problem here would be that the text cell will grow in height when one character is typed in (which we don't want ideally). Can you take a look at that?

**Comment by MaddyDev** [comment]
> @MaddyDev I believe the problem here would be that the text cell will grow in height when one character is typed in (which we don't want ideally). Can you take a look at that?

Not sure I understand the issue you're referring above, but I tested typing only one character and the experience still looks the same.

**Comment by chlafreniere** [comment]
Just tested it and you're right. I swear we looked at this very fix previously. Oh well 😄. Thanks!

**Review by mutza974 — CHANGES_REQUESTED** [review]
Thanks

**Comment by mutza974** [comment]
Duplicate of #

File: src/sql/workbench/contrib/notebook/browser/cellViews/textCell.css [hunk 1]
```diff
@@ -17,6 +17,8 @@ text-cell-component .notebook-preview {
 	flex-direction: column;
 	padding-left: 8px;
 	padding-right: 8px;
+	padding-top: 14px;
```
Comment (mutza974) [inline src/sql/workbench/contrib/notebook/browser/cellViews/textCell.css:20]: Thanks

File: src/sql/workbench/contrib/notebook/browser/cellViews/textCell.css [hunk 2]
```diff
@@ -17,6 +17,8 @@ text-cell-component .notebook-preview {
...
+	padding-bottom: 14px;
```
Comment (mutza974) [inline src/sql/workbench/contrib/notebook/browser/cellViews/textCell.css:21]: Thanks

**Review by mutza974 — CHANGES_REQUESTED** [review]
@mutza974

**Review by chlafreniere — APPROVED** [review]

**Review by barbaravaldez — CHANGES_REQUESTED** [review]
the size of the cell increases by 14px, when typing in Split View
Screen Shot 2020-11-02 at 11 06 30 AM

**Comment by MaddyDev** [comment]
> the size of the cell increases by 14px, when typing in Split View
> Screen Shot 2020-11-02 at 11 06 30 AM

Am not sure why we should have the padding only while editing and not in preview mode. @chlafreniere Is this fine? Or should I add styles to remove the padding while in preview mode?

**Comment by MaddyDev** [comment]
Updated show-preview style to remove the added padding so it retains the previous behavior. Below is how it looks when you type:
image

**Review by barbaravaldez — APPROVED** [review]

**Pull request merged**